src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -66,6 +71,23 @@ public static boolean isNonZeroUnsignedInteger(String s) {
         }
     }
 
+    /**
+     * Validates whether {@code score} is a valid integer.
+     *
+     * @param score the string representation of the score under check.
+     * @throws ParseException if the score is not a valid integer.
+     */
+    public static void validateScore(String score) throws ParseException {
+        if (score.equals("")) {
+            throw new ParseException(MESSAGE_INVALID_COMMAND_FORMAT);
+        }
+        try {
+            int scoreValue = Integer.parseInt(score);
```

Comment:
Just to check, score can be negative as well right

Reply:
nope it cant
In the score class we set a restriction for it to be between 0 and 100